airbyte-integrations/connectors/source-sentry/source_sentry/manifest.yaml

```diff
@@ -1,105 +1,175 @@
-      type: "CursorPagination"
-      cursor_value: "{{ headers.link.next.cursor }}"
-      stop_condition: "{{ headers.link.next.results != 'true' }}"
+      type: CursorPagination
+      cursor_value: '{{ headers["link"].get("next", {}).get("cursor", {}) }}'
+      stop_condition: '{{ headers["link"]["next"]["results"] == "false" }}'
   retriever:
     type: SimpleRetriever
-
-streams:
-  - type: DeclarativeStream
-    $parameters:
-      # https://docs.sentry.io/api/events/list-a-projects-events/
-      name: "events"
-    primary_key: "id"
+    requester:
+      $ref: "#/definitions/requester"
+    record_selector:
+      $ref: "#/definitions/record_selector"
+    paginator:
+      $ref: "#/definitions/paginator"
+    partition_router: []
+  retriever_with_filter:
+    type: SimpleRetriever
+    requester:
+      $ref: "#/definitions/requester"
+    record_selector:
+      $ref: "#/definitions/record_selector"
+      record_filter:
+        condition: "{{ record[parameters['cursor_field']] > stream_state.get(parameters['cursor_field'], '') }}"
+    paginator:
+      $ref: "#/definitions/paginator"
+    partition_router: []
+  incremental_sync:
+    type: DatetimeBasedCursor
+    cursor_field: "{{ parameters['cursor_field'] }}"
+    cursor_datetime_formats:
+      - "%Y-%m-%dT%H:%M:%S.%f%z"
+      - "%Y-%m-%dT%H:%M:%SZ"
+    datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+    start_datetime:
+      type: MinMaxDatetime
+      datetime: "1900-01-01T00:00:00.0Z"
+      datetime_format: "%Y-%m-%dT%H:%M:%S.%fZ"
+    is_data_feed: true
+  base_stream_full_refresh:
     schema_loader:
       $ref: "#/definitions/schema_loader"
     retriever:
       $ref: "#/definitions/retriever"
-      record_selector:
-        $ref: "#/definitions/selector"
+  base_stream_incremental:
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/retriever_with_filter"
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Events https://docs.sentry.io/api/events/list-a-projects-error-events/
+  events:
+    type: DeclarativeStream
+    $parameters:
+      name: "events"
+      primary_key: "id"
+      path: "projects/{{ config['organization'] }}/{{ config['project'] }}/events/"
+      cursor_field: "dateCreated"
+    retriever:
+      type: SimpleRetriever
       requester:
         $ref: "#/definitions/requester"
-        path: "projects/{{config.organization}}/{{config.project}}/events/"
         request_parameters:
           full: "true"
+      record_selector:
+        $ref: "#/definitions/record_selector"
+        record_filter:
+          condition: "{{ record[parameters['cursor_field']] > stream_state.get(parameters['cursor_field'], '') }}"
       paginator:
         $ref: "#/definitions/paginator"
-  - type: DeclarativeStream
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Issues https://docs.sentry.io/api/events/list-a-projects-issues/
+  issues:
```

Comment:
Otherwise now all users have stats fields in their data and this field is null due to `statsPeriod=''` in request. The only difference in data is that previous records don't have stats in response and current response do have stats. Maybe we should remove `statsPeriod=''` from request in low code to fill stats in users destinations?

Reply:
It seems like "more data == better" but I don't know the specifics of Sentry. Without knowing this, I would:
* See who added `statsPeriod` and ask them what was the reasoning
* If we can't do the first point, YOLO it and remove `statsPeriod`

Reply:
Ok, I'm fine with this decision. Is there somewhere we can document this? Changelog is a bit too restrictive in terms of formatting. Maybe the README.md or this PR's description?